etc/systemd/iscsi-init.service.template

```diff
@@ -16,4 +16,4 @@ WantedBy=systemd-remount-fs.service
 [Service]
 Type=oneshot
 RemainAfterExit=no
-ExecStart=/usr/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
+ExecStart=/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
```

Comment:
Why do you think "/bin/sh" in the *only* valid name? And you misspelled "name" in your commit message.

Reply:
If you want a citation, then https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s04.html or https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s04.html#specificOptions11 (alas, it's FHS, oh well).

What's harder to cite is that that's because (barring the anomaly that was SVr4 in particular with `/sbin/sh`), since V1, that's *the* location of the shell.

/usr/bin/sh is only valid on merged-/usr systems (in which case /bin *is* /usr/bin). Trivially, not all systems are merged-/usr.

Reply:
Okay, I'll accept that then.